src/chrome/content/rules/Freifunk_Gera-Greiz.xml

```diff
@@ -0,0 +1,17 @@
+<ruleset name="Freifunk Gera-Greiz">
+	<target host="freifunk-gera-greiz.de"/>
+	<target host="www.freifunk-gera-greiz.de"/>
```

Comment:
only one of them listens on the port 443 and has a valid SSL certificate and I can't change that.